sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -17,16 +17,15 @@
 #include <cstdarg>
 #include <cstdio>
 #include <cstring>
+#include <level_zero/zes_api.h>
```

Comment:
Why did you move these?

Reply:
I ran clang-format to the file before uploading the patch, it should be moved by the format tool.
Thanks very much.

Reply:
can you please put them back, perhaps using comments that shut of clang-format
it is weird to have them in the middle of STL includes

Reply:
Hi, @smaslov-intel 
I just reverted the unnecessary header file change, could you please help approve again?
Thanks very much.